eth-riscv-syscalls/src/lib.rs

```diff
@@ -50,11 +50,14 @@ macro_rules! syscalls {
 // t0: 2, opcode for sstore, a0: storage key, a1: storage value, returns nothing
 // t0: 3, opcode for call, args: TODO
 // t0: 4, opcode for revert, doesn't return
+// t0: 5, opcode for caller, returns an address
+// t0: 6, opcode for keccak256, a0: offset, a1: size, returns keccak256 hash
 syscalls!(
     (0, Return, "return"),
     (1, SLoad, "sload"),
     (2, SStore, "sstore"),
     (3, Call, "call"),
     (4, Revert, "revert"),
     (5, Caller, "caller"),
+    (6, Keccak256, "keccak256"),
```

Comment:
Let's start using EVM opcodes as syscall ids here

Reply:
@leonardoalt let's do on both PRs (for log also) with opcodes and then in a new PR/commit the rest